hardware/tv-hat/README.md

```diff
@@ -0,0 +1,58 @@
+## Getting started with the Raspberry Pi TV HAT
+
+The TV HAT allows you to receive and decode digital television streams on your Raspberry Pi through its on-board DVB-T2 tuner. Then you can watch these streams on any computer connected to the same network as the Pi.
+
+The software we recommend to decode the streams (known as multiplexes, or muxes for short) and view content is called TVHeadend, and instructions for setting it up are below. The TV HAT can decode one mux at a time, and each mux can contain several channels to choose from. Content can either be viewed on the Raspberry Pi to which the TV-HAT is connected, or sent to another device on the same network.
+
+**You will need:**
+* A TV aerial
+* A Raspberry Pi TV HAT with its stand-offs, screws, and aerial adaptor
+* A Raspberry Pi that is connected to the internet (plus a mouse, keyboard, and display, if
+you are not accessing it remotely)
+* Another computer connected to the same network
```

Comment:
Is this last item optional, if viewing TV on the Pi itself?

Reply:
TVHeadend -> tvheadend per their website.